spec/models/mms_client_spec.rb

```diff
@@ -11,28 +11,33 @@
     end
 
     it 'makes a request to get MODS' do
-      expect(HTTP).to receive(:get).with('http://example.com/exports/mods/abc-123') { HTTP }
+      expect(HTTP).to receive(:get).with('http://example.com/exports/mods/abc-123', params: {}) { double(code: 200) }
       @mms_client.mods_for('abc-123')
     end
 
     it 'makes a request to get RELS-EXT' do
-      expect(HTTP).to receive(:get).with('http://example.com/exports/rels_ext/abc-123') { HTTP }
+      expect(HTTP).to receive(:get).with('http://example.com/exports/rels_ext/abc-123', params: {}) { double(code: 200) }
       @mms_client.rels_ext_for('abc-123')
     end
 
     it 'makes a request to get Rights' do
-      expect(HTTP).to receive(:get).with('http://example.com/exports/rights/abc-123') { HTTP }
+      expect(HTTP).to receive(:get).with('http://example.com/exports/rights/abc-123', params: {}) { double(code: 200) }
       @mms_client.rights_for('abc-123')
     end
 
     it 'makes a request to get Dublin Core' do
-      expect(HTTP).to receive(:get).with('http://example.com/exports/dc/abc-123') { HTTP }
+      expect(HTTP).to receive(:get).with('http://example.com/exports/dc/abc-123', params: {}) { double(code: 200) }
       @mms_client.dublin_core_for('abc-123')
     end
 
     it "makes a request to get an Item's Captures" do
-      expect(HTTP).to receive(:get).with('http://example.com/exports/get_captures/abc-123', params: {showAll: 'true'}) { HTTP }
+      expect(HTTP).to receive(:get).with('http://example.com/exports/get_captures/abc-123', params: {showAll: 'true'}) { double(code: 200) }
       @mms_client.captures_for_item('abc-123')
     end
+
+    it "Throws exceptions on bad requests" do
```

Comment:
Do you see what happens Larry?
Do you see what happens when you make a request that doesn't have a mocked response, Larry?
Do you see what happens when make a request that's bound to fail.